src/config.rs

```diff
@@ -448,6 +458,16 @@ impl Config {
     pub fn default_path() -> String {
         String::from("pgcat.toml")
     }
+
+    pub fn overwrite_passwords(&mut self) {
+        if let Ok(val) = env::var("PGCAT_ADMIN_PASSWORD") {
```

Comment:
We don't allow env-based configuration at the moment, only file based.

Reply:
I added it because it goes aside with the `auth_query` thing. One of the things that we get from the `auth_query` is being able to stop using cleartext passwords on the config. To really get that done, we need a way of setting these two passwords so I added a simple mechanism for overwriting them using ENV vars.

That said, the actual reason (I have) is being able to deploy to kubernetes without losing the `auto reload` feature, which I find neat.

Normally, secrets are set as ENV vars in k8s. If I have no way of setting that password through and ENV var, I should create an script that generates it at boot time. The script should, provided a config file with a placeholder for those passwords, substitute those using the env vars, and write to a file that will be fed to pgcat. 

The problem is that then (apart from this nighmare), I lose the auto reload feature, well not really loose it, but it would make me deploy the thing using a sidecar that does the password rewrite every time the templated config changes, so later pgcat sees it and does the actual reload.

I find this a bit cumbersome because it complicates containerized deploys for pgcat (the same would apply to different orchestrators) so I was hoping that these two variables could be an exception.